projects/ng-components/lib/table/table-td/table-td.component.html

```diff
@@ -1,8 +1,8 @@
 <ng-container [ngSwitch]="valueType">
-    <span *ngSwitchCase="'string'">{{ row }}</span>
-    <span *ngSwitchCase="'number'">{{ row | number }}</span>
-    <span *ngSwitchCase="'datetime'">{{ row | date: dateFormat + " HH:mm:ss" }}</span>
-    <span *ngSwitchCase="'date'">{{ row | date: dateFormat }}</span>
-    <span *ngSwitchCase="'custom-html'" [innerHtml]="row | safeHtml"></span>
+    <span *ngSwitchCase="'string'"> {{ row || "–" }}</span>
```

Comment:
"–" should be possible to configure, not all the time its a simple string, sometimes it should be empty or in my case it has to be a tooltip, would it be possible to add some extra Input to enable this kind of feature ?

Reply:
Would `custom-html` suffice for your example or do you want to be able to set your own string to display if the row is empty?